frontend/src/pages/GettingStarted.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import Buttons from '../lib/Buttons';
+import { Page } from './Page';
+import { ToolbarProps } from '../components/Toolbar';
+import { classes } from 'typestyle';
+import { commonCss, padding } from '../Css';
+
+export class GettingStarted extends Page<{}, {}> {
+  public getInitialToolbarState(): ToolbarProps {
+    const buttons = new Buttons(this.props, this.refresh.bind(this));
+    return {
+      actions: buttons.getToolbarActionMap(),
+      breadcrumbs: [],
+      pageTitle: 'Getting Started',
+    };
+  }
+
+  public async refresh() {
+    // do nothing
+  }
+
+  public render(): JSX.Element {
+    return (
+      <div className={classes(commonCss.page, padding(20, 'lr'))}>
```

Comment:
Thank you so much for all the work ! this is very nicely done and an amazing turn around time!
Do you think there is any chance that you could match the lay out in option 0 ("in Templates, Demos ,and Tutorials UI design" for the content of this page.  if not no worries you already have done most of the heavy lifting,  I'm just worried about the short time remaining tomorrow.

Reply:
Great, I'm glad it's helpful.
Also updated the start page to use markdown with some styling I just defined. I believe that will be the most easy to change for the team later.

Reply:
This is perfect ! thank you .